src/Microsoft.AspNet.Authentication/AuthenticationServiceCollectionExtensions.cs

```diff
@@ -12,6 +12,7 @@ public static class AuthenticationServiceCollectionExtensions
     {
         public static IServiceCollection AddAuthentication([NotNull] this IServiceCollection services)
         {
+            services.AddCaching();
```

Comment:
No, we don't want to add caching services and dependencies for all auth providers, just OIDC.